lib/ug.js

```diff
@@ -1,19 +1,33 @@
 const { Client } = require('ldapts')
 const log = require('skog')
 
+let ldapClient
+
+async function ldapBind () {
+  log.info('Connecting to UG via LDAP...')
+  ldapClient = new Client({
+    url: process.env.UG_URL
+  })
+  await ldapClient.bind(process.env.UG_USERNAME, process.env.UG_PASSWORD)
+}
+
+async function ldapUnbind () {
+  log.info('Disconnecting to UG via LDAP...')
+  await ldapClient.unbind()
+}
+
 async function ldapSearch ({
   base = 'OU=UG,DC=ug,DC=kth,DC=se',
   filter = '',
   attributes = [],
   scope = 'sub'
 }) {
-  let ldapClient
+  if (!ldapClient) {
```

Comment:
Either check if ldapClient is bound, remove the variable on unBind, or remove this check. At the moment this check is faulty